dynamic_rest/viewsets.py

```diff
@@ -337,4 +336,90 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_PARTIAL_CREATION = settings.ENABLE_BULK_PARTIAL_CREATION
+
+    def _create_many(self, data):
+        items = []
+        errors = []
+        result = {}
+        serializers = []
+
+        for entry in data:
+            serializer = self.get_serializer(data=entry)
+            try:
+                serializer.is_valid(raise_exception=True)
+            except exceptions.ValidationError as e:
+                errors.append({
+                    'detail': str(e),
+                    'source': entry
+                })
+            else:
+                if self.ENABLE_BULK_PARTIAL_CREATION:
+                    self.perform_create(serializer)
+                    items.append(serializer.data)
+                else:
+                    serializers.append(serializer)
+        if not self.ENABLE_BULK_PARTIAL_CREATION and not errors:
+            for serializer in serializers:
+                self.perform_create(serializer)
+                items.append(serializer.data)
```

Comment:
I noticed there's a bit of redundancy in the output that's caused by the use of `serializer.data`, which wraps the result of `serializer.to_representation`:

```
{
  'groups': [
    {
      'group': {
          'users': [],
          'name': u'bar',
          'loc1usersLambda': [], 'members': [], 'loc1users': [], 'id': 2, 'permissions': []
      }
    }
  ]
  'errors': [
    {
      'detail': "{'name': [u'Group with this name already exists.']}",
      'source': {u'name': u'foo'}
    }
  ],
}
```

Rather than returning a payload that includes "groups" at the top-level, and also the "group" singular name for each group, we should just return "groups" with an array, as we would if we were doing a GET request for groups. 

In other words, it should look like this:

```
{
  'groups': [
    {
        'users': [],
        'name': u'bar',
        'loc1usersLambda': [], 'members': [], 'loc1users': [], 'id': 2, 'permissions': []
    }
  ]
  'errors': [
    {
      'detail': "{'name': [u'Group with this name already exists.']}",
      'source': {u'name': u'foo'}
    }
  ],
}
```

You can accomplish this by replacing `serializer.data` with `serializer.to_representation(serializer.instance)`